index.js

```diff
@@ -1,8 +1,8 @@
 'use strict'
 
-var PassThrough = require('readable-stream').PassThrough
-var inherits = require('inherits')
-var p = require('process-nextick-args')
+const { PassThrough } = require('readable-stream')
```

Comment:
Use of `const` might mean loss of browser support (not that there was any official support).

Reply:
There wasn't any. If there is, I'm happy to revisit.

Reply:
Are you using this in the browser?

Reply:
No, I'm not. Was just an observation :) It's fine, esp. now that it's a conscious decision